src/main/java/com/epam/tc/service/TrelloService.java

```diff
@@ -0,0 +1,134 @@
+package com.epam.tc.service;
+
+import com.epam.tc.dto.BoardDto;
+import com.epam.tc.dto.LabelDto;
+import com.epam.tc.dto.ListDto;
+import com.epam.tc.utils.ConfProperties;
+import com.epam.tc.utils.EndPoints;
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.specification.RequestSpecification;
+import lombok.NoArgsConstructor;
+import org.json.simple.JSONObject;
+
+import static org.apache.http.HttpStatus.SC_OK;
+
+@NoArgsConstructor
+
+public class TrelloService {
```

Comment:
There are some code duplications in this class, could you probably extract some common parts?

Reply:
Fix this